constantinCurtis/src/main/java/data/collection/task/classes/Order.java

```diff
@@ -0,0 +1,52 @@
+package data.collection.task.classes;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Order {
+    private static long idCounter = 0;
+    public long id;
+    public LocalDate date;
+    public User user;
+    public List<Item> items;
+
+    //cntrl+alt+l =arrangare de cod
```

Comment:
nice try...This is a part of your application? Please, store it somewhere else and let the Order.java contain only class related code.

Reply:
deleted the row 14